populate/src/main/java/bio/terra/pearl/populate/service/SurveyPopulator.java

```diff
@@ -81,10 +81,7 @@ public Optional<Survey> findFromDto(SurveyPopDto popDto, PortalPopulateContext c
 
     @Override
     public Survey overwriteExisting(Survey existingObj, SurveyPopDto popDto, PortalPopulateContext context) throws IOException {
-        existingObj.setContent(popDto.getContent());
-        existingObj.setName(popDto.getName());
-        existingObj.setFooter(popDto.getFooter());
-        existingObj.setPublishedVersion(popDto.getPublishedVersion());
+        BeanUtils.copyProperties(popDto, existingObj, "id", "stableId", "version", "createdAt", "lastUpdatedAt", "answerMappings", "portalId");
```

Comment:
maybe i'm confused, but doesn't it also need to copy `content`, `name`, `footer` and `publishedVersion`?

Reply:
we do, the syntax of 'copyProperties' is that it copies all properties *except* the ones explicitly listed

Reply:
Ahhhhh, that is a confusing function haha